packages/rocketchat-ui/client/views/app/room.js

```diff
@@ -50,6 +50,14 @@ const openProfileTabOrOpenDM = (e, instance, username) => {
 };
 
 Template.room.helpers({
+	avatarUrl() {
```

Comment:
You're returning an avatar background and not the avatar url, change the name or return only the url :)